docs/docsite/rst/roadmap/COLLECTIONS_8.rst

```diff
@@ -0,0 +1,57 @@
+.. _ansible_8_roadmap:
+
+===================
+Ansible project 8.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See the `ansible-core 2.15 Roadmap <https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_15.html>`_ for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2023-03-31: ansible-core feature freeze, stable-2.15 branch created.
+:2023-04-03: Start of ansible-core 2.15 betas (biweekly, as needed).
+:2023-04-04: Ansible-8.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.15`` pre-releases).
+:2023-04-19: Community topic: List any backwards incompatible collection releases that beta1 should try to accommodate.
+:2023-04-24: First ansible-core 2.15 release candidate.
+:2023-05-02: Ansible-8.0.0 alpha2.
+:2023-05-03: Community Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
+:2023-05-15: Ansible-core-2.15.0 released.
+:2023-05-15: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-8. This includes adding new collections to Ansible 8.0.0; from now on new collections have to wait for 8.1.0 or later.
+:2023-05-16: Create the ansible-build-data directory and files for Ansible-8.
```

Comment:
We only did this when GA happened for the last few releases, since otherwise the changelog ancestor will have to be adjusted again once GA happened. Also, it should say Ansible-9, since that stuff for Ansible-8 is already there (even now).
```suggestion
```